README.md

```diff
@@ -212,7 +212,7 @@ nukkit {
     prefix = 'TEST'
     
     commands {
-        test {
+        'test' {
```

Comment:
Hmm, what's the problem with this? The same is suggested for Bukkit so apparently it has worked at some point :)

Reply:
I was under the impression you had to use strings here, unless Gradle/Groovy does some magic I'm unaware of?

Reply:
Well, Groovy contains loooooooooooooooooooooooooooots of magic :D

Reply:
Would you like me to just revert that change, in that case?

Reply:
I was hoping you could confirm that this indeed fixes something that is broken :)

Reply:
Seems it doesn't from a test. Groovy sure is funky... I'll revert that.